lib/models/users.js

```diff
@@ -19,18 +20,44 @@ module.exports = function (sequelize, DataTypes) {
             },
             email: {
                 type: DataTypes.STRING(32),
-                allowNull: false
-            },
-            createdAt: {
-                field: 'created_at',
-                type: DataTypes.DATE,
                 allowNull: false,
-                defaultValue: sequelize.literal('CURRENT_TIMESTAMP')
+                unique: 'email'
+            },
+            mobile_no: {
```

Comment:
can we move to camelCase instead of snake_case

just mention field: 'mobile_no' instead

Reply:
Done.